src/components/todos/todo-list-create-dialog.tsx

```
@@ -38,6 +42,35 @@ export function TodoListCreateDialog({ isOpen, onClose }: TodoListCreateDialogPr
     );
   };
 
+  const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    if (file) {
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)